pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/NamespaceStatsAggregator.java

```diff
@@ -104,6 +115,9 @@ public static void generate(PulsarService pulsar, boolean includeTopicMetrics, b
         if (includeTopicMetrics) {
             printTopicsCountStats(stream, localNamespaceTopicCount, cluster);
         }
+
+        //Print broker level metrics to prometheus
```

Comment:
I would omit that comment line, as the method's name suffices here to clarify the intent.

Reply:
This is a good suggestion

Reply:
> I would omit that comment line, as the method's name suffices here to clarify the intent.

This is a good suggestion